extras/help_generator/writers.py

```diff
@@ -184,13 +184,14 @@ def write_helpindex(helpdir):
         html_list.append('</tr></table></center>')
         html_list.append('<center><table class="commands" id="%s">'
                          % doubles[0])
-        for item in filelist:
+        keyfunc = lambda x: x.lower().rsplit('/', 1)[1]
+        for item in sorted(filelist, key=keyfunc):
```

Comment:
The main reason to do it like this was to respect the line length limit of PEP8. But you are right with the lambda and I've factored `keyfunc` out into a proper function in ae6880b.